### DISCUSSION TITLE
Ripple Effect

### DISCUSSION THREAD

**Discussion by TiagoPerdigao1983** [rsuite/rsuite#1639] [Q&A]
Hi, 

How would I remove the onclick ripple effect when I click an element on the sidenav?

Thank you for your help.

**Comment by simonguo** [comment]
The style can be overridden by setting the less variable.
```less
@button-ripple: false;
```
https://rsuitejs.com/guide/themes

**Reply by TiagoPerdigao1983** [reply]
Thank you. Tried that with create react app and was unable to implement it. My solution was with the css: 
`.rs-ripple-pond {
	display: none !important;
}`
